### PR TITLE
[plugin.video.currenttime.tv] 1.0.1 (wrong branch)

### PR DESCRIPTION
**Pull request by dicoka**
### Description

Plugin allows watching live stream, video and broadcasts archive for http://currenttime.tv - new TV channel of Voice of America (VOA) and Radio Free Europe/Radio Liberty (RFE/RL) for Russian-speaking audiences. With 'Current Time' the networks are offering audiences professional, independent journalism that they cannot get elsewhere - regional and global issues that receive little coverage on official (propaganda) channels.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.currenttime.tv/addon.py [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    [30037, 30038, 'lastvids+archive',  'nvamerica',    '/z/20347.html'],
+    [30039, 30040, 'lastvids+archive',  'oba', 		    '/z/20366.html'],
+    [30041, 30042, 'lastvids+archive',  'itogi', 	    '/z/17499.html'],
+    [30043, 30044, 'lastvids+archive',  'week',		    '/z/17498.html'],
+    [30045, 30046, 'lastvids+archive',  'baltia', 	    '/z/20350.html'],
+    [30047, 30048, 'lastvids+archive',  'bisplan', 	    '/z/20354.html'],
+    [30049, 30050, 'lastvids+archive',  'unknownrus',   '/z/20331.html'],
+    [30051, 30052, 'lastvids+archive',  'guests', 	    '/z/20330.html']
+])
+default_view = {
```
Comment (phil65) [inline plugin.video.currenttime.tv/addon.py]: please get rid of this viewtype override.

File: plugin.video.currenttime.tv/addon.py [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+folder_title = args.get('title', [None])[0]
+folder_level = int(args.get('level', '0')[0])
+folder_name = args.get('name', [None])[0]
+
+site_url = 'http://www.currenttime.tv'
+addon_name = 'plugin.video.currenttime.tv'
+addon = xbmcaddon.Addon(addon_name)
+addon_icon = addon.getAddonInfo('icon')
+
+xbmcplugin.setContent(addon_handle, 'tvshows')  # !!!
```
Comment (phil65) [inline plugin.video.currenttime.tv/addon.py]: "videos" might make more sense?

File: plugin.video.currenttime.tv/addon.py [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+try:
+    ### Main menu
+    if mode is None:
+        mode = ['main_menu']
+        add_dir({
+            'name':     'live',
+            'thumb':    img_link('live', 'thumb'),
+            'fanart':   img_link('live', 'fanart'),
+            'mode':     'play',
+            'title':    '[COLOR blue]' + addon.getLocalizedString(30001).encode('utf-8') + '[/COLOR]',
```
Comment (phil65) [inline plugin.video.currenttime.tv/addon.py]: no label coloring, please.

File: plugin.video.currenttime.tv/addon.py [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+            if video_pages_buffer[lnum] is not None:
+                add_dir(get_video_dir(video_pages_buffer[lnum]))
+        # add menu item 'Next'
+        if llast < len(match_url):
+            add_dir({
+                'name':     folder_name,
+                'thumb':    img_link('folder', 'thumb'),
+                'fanart':   img_link(folder_name, 'fanart'),
+                'mode':     'lastvids+next',
+                'title':    '[B][COLOR blue]>> ' + addon.getLocalizedString(30101).encode('utf-8')
```
Comment (phil65) [inline plugin.video.currenttime.tv/addon.py]: same here.

File: plugin.video.currenttime.tv/resources/language/English/strings.po [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+"Last-Translator: dicoka@ukr.net\n"
+"Language-Team: dicoka@ukr.net\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+msgctxt "#30001"
+msgid "[B]Live Stream[/B]"
```
Comment (phil65) [inline plugin.video.currenttime.tv/resources/language/English/strings.po]: formatting shouldnt be part of language file.

**Comment by phil65** [comment]
Apart from my comments, please PR to correct branch. (judging from your addon.xml it should be branch "krypton")

File: plugin.video.currenttime.tv/addon.py (hunk 2 above)
Comment (romanvm) [inline plugin.video.currenttime.tv/addon.py]: Is "videos" string a valid content type? It's not mentioned in the docs: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcplugin.html#gaa30572d1e5d9d589e1cd3bfc1e2318d6

**Pull request closed without merging**